packages/desktop-client/src/components/accounts/Account.jsx

```diff
@@ -275,6 +276,8 @@ class AccountInternal extends PureComponent {
     this.unlisten = () => {
       unlistens.forEach(unlisten => unlisten());
     };
+    
+    this.applyFilters(this.state.conditions);
```

Comment:
The second [fetchTransaction](https://github.com/actualbudget/actual/pull/2408/files#diff-21c5b30e746cbe9a3777fbd9c245d00284320a22e7a18e6c443c2466d1b97767R285)below will cause an unnecessary render because applyFilter already calls it

Reply:
applyFilters only calls fetchTransactions if conditions.length <= 0 which is not ture for this case

Reply:
That is true, but it will all boil down to updateQuery which updates the state and triggers a render. There will 2 calls to updateQuery (one from applyFilters and another from fetchTransactions), hence 2 renders.

Reply:
If I leave applyfilters and remove the fetch transactions (below) the table does not render any rows (even when state.conditions exists)

Reply:
Looks like applyFilters depend on the rootQuery being initialized. Maybe we can pass the conditions to fetchTransactions and then call applyFilters (after the rootQuery is initialized) when conditions parameter exists, otherwise just call the updateQuery as before. Did a bit of testing and seems to solve the flickering issue.

Reply:
That's golden. Thanks for the assist. Let me know if I've implemented how you imagined. Cheers!